### PR TITLE
Added labels equatable check to ListDiffable isEqual method

### LINKED ISSUE #715: Issue label occasionally doesn’t appear visible
<details>
<summary>Bug Report Dump (Auto-generated)</summary>
<pre>
Version 1.12.0 (1079)
Device: iPhone 7 (iOS 11.0)
TestFlight: true
</pre>
</details>

### PR DESCRIPTION
**Pull request by joerocca**
Fixes #715

### CODE REVIEW

**Comment by heshamsalman** [comment]
@BasThomas that 4.1 change is _doooooope_

**Comment by rnystrom** [comment]
omfg, pls

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by BasThomas** [comment]
And it's getting even better, potentially! https://github.com/apple/swift/pull/12598

File: Classes/Repository/RepositoryIssueSummaryModel.swift [hunk 1]
```diff
@@ -56,5 +56,6 @@ class RepositoryIssueSummaryModel: ListDiffable {
             && author == object.author
             && created == object.created
             && title.attributedText.string == object.title.attributedText.string
+            && labels == object.labels
```
Comment (rnystrom) [inline Classes/Repository/RepositoryIssueSummaryModel.swift]: Note that this check is `O(n)` (comparing all contents of an array). I'm going to clean this up to be much faster.

**Pull request merged**